site/src/_static/chart@3.9.1.min.js

```diff
@@ -0,0 +1,15 @@
+/*!
```

Comment:
Alternative to including static libs in our repo is to NPM install it and then copy it from `node_modules` to `_static` as part of the build step.